### ISSUE TITLE
Double Letter Typing

### ISSUE THREAD

**Issue by HamNCheeseBorger** [CampbellCrowley/VRC-UdonSharp-Keyboard#1]
The letters output in the Enter Text and on the screen twice when clicking a button once. Its a cool keyboard project and im sure the fix could be doable on my own but I wanted this post to be made so anyone who has this issue has somewhere to look.

**Comment by CampbellCrowley** [comment]
Thanks for bringing this up!
Are you testing in-game or in Unity at this point?

I'm having trouble reproducing this in-game, but I am able to see this behavior in the Client Simulator within Unity.
For me it's clicking each button 3 times.

I'm not sure how to fix this from happening in the Client Sim, it might be a bug on the VRC team's side of things. It seems like their Input manager (ClientSimInputModule maybe) may be firing the "Pointer Down" event multiple times.

**Comment by HamNCheeseBorger** [comment]
Thanks for the response, I am testing with the client, odd for me its just double lettering. In game it oddly doesn't work at all. I cant interact with the keyboard. Its my first time making worlds, familiar with unity, pretty good avatar making skills, and beginner at C# but completely new to world building. So im kinda just throwing stuff in and learning how it works. 

Tomorrow ill try a blank world with just the keyboard and like the world SDK see if it is the same issue as yours. I hope you figure this out cause its a pretty cool prefab! Keep up the good work

**Comment by CampbellCrowley** [comment]
I don't believe the duplicate letter typing is an issue with this keyboard. I've opened a bug report on VRChat's Canny here: https://feedback.vrchat.com/bug-reports/p/ui-pointer-down-triggering-multiple-times-per-click-in-unity-only

I did notice however too, in your case you only get the keypress event 2 times per click, which for me happened if the keyboard was on the UI layer in Unity instead of Default. So if you're still having issues getting it to work in game, perhaps that would be something to check.
